app/models/flexible_page_types/history_page.json

```diff
@@ -1,3 +1,31 @@
 {
-  "key": "history_page"
+  "key": "history_page",
+  "schema": {
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://www.gov.uk/schemas/history_page/v1",
+    "title": "History page",
+    "description": "A history page on GOV.UK",
+    "type": "object",
+    "properties": {
+      "page_title": {
+        "title": "Page title",
+        "description": "The title of the history page",
+        "type": "object",
+        "properties": {
+          "heading_text": {
+            "title": "Heading text",
+            "description": "The main heading text for the page title",
+            "type": "string"
+          },
+          "context": {
+            "title": "Context",
+            "description": "Some additional context for the page title",
+            "type": "string"
+          }
```

Comment:
I think we're missing the `lead_paragraph` property?  See https://github.com/alphagov/govuk-rfcs/pull/182/files#diff-98512ba43959e2b34beee8dfddb363eb84dcfbac857576b27ecf04355fef21e3R135